### PR TITLE
[ADD] Quality: Pass - Fail quality checks

### PR DESCRIPTION
**Pull request by hojo-odoo**
This PR adds a new doc detailing how to create and process Pass - Fail quality checks.

NOTE: One of the checks is failing because the PR contains a reference to the "Quality checks" doc, which is not yet live. That doc should be live before this one, at which point the ref tags in this doc will work as intended.

### CODE REVIEW

**Comment by StraubCreative** [comment]
@odoo/us-doc-review 
Need #6454 to merge first, which includes the necessary toctree updates for the CI checks to pass

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  check and the criteria that must be met for the check to pass.
+
+.. image:: pass_fail_check/qcp-form.png
+   :align: center
+   :alt: A Quality Control Point (QCP) form configured to create a Pass - Fail quality check.
+
+Process a Pass - Fail quality check
+===================================
+
+Once created, there are multiple ways that *Measure* quality checks can be processed. If a quality
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:63]:
```suggestion
Once created, there are multiple ways that *Pass - Fail* quality checks can be processed. If a quality
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+check is assigned to a specific inventory, manufacturing, or work order, the check can be processed
+on the order itself. Alternatively, a check can be processed from the check's page.
+
+From the check's page
+---------------------
+
+To process a *Measure* quality check from the check's page, begin by navigating to
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:70]:
```suggestion
To process a *Pass - Fail* quality check from the check's page, begin by navigating to
```

Comment (Felicious) [reply content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:70]: John, I want to show you a neat lil trick in VS Code for finding and replacing instances of a word.

1. Click CTRL + F to open the word finder at the top right of your screen
image
2. Enter the word you'd like to find, i.e. "Measure" and it'll show u 
image
3. You can manually modify it, but to automate it, click the little downwards arrow in the word finder window, and type the word you'd like to replace "Measure" with
image
4. Then, in the replace text field, type "Pass - Fail" right now, and click enter to replace the first instance of Measure, highlighted in bright colors
image

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+Fail* quality check step is reached. Upon reaching the check, follow the instructions that appear at
+the top of the screen. If the criteria for the check is met, click the :guilabel:`Pass` button at
+the top-right of the screen. If the criteria is not met, click the :guilabel:`Fail` button.
+
+.. image:: pass_fail_check/work-order-pass-fail-check.png
+   :align: center
+   :alt: A Pass - Fail check for a manufacturing work order.
+
+If a quality alert must be created, do so by clicking the :guilabel:`☰ (menu)` button in the tablet
+view, and selecting :guilabel:`Quality Alert` from the :guilabel:`Menu` pop-up window. A
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:129]:
```suggestion
If a quality alert must be created, do so by clicking the :guilabel:`☰ (hamburger)` button in the
tablet view, and selecting :guilabel:`Quality Alert` from the :guilabel:`Menu` pop-up window. A
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+tab, and then clicking the :guilabel:`📱 (tablet)` button on the order's line.
+
+With tablet view open, complete the steps listed on the left side of the screen until the *Pass -
+Fail* quality check step is reached. Upon reaching the check, follow the instructions that appear at
+the top of the screen. If the criteria for the check is met, click the :guilabel:`Pass` button at
+the top-right of the screen. If the criteria is not met, click the :guilabel:`Fail` button.
+
+.. image:: pass_fail_check/work-order-pass-fail-check.png
+   :align: center
+   :alt: A Pass - Fail check for a manufacturing work order.
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:126]: Yay! I appreciate that you applied my suggestion for a previous PR in this one too :smile:

**Review by Felicious — APPROVED** [review]
This is a really really solid doc, @hojo-odoo :smile: 

It's concise and really easy to follow. I just had some minor typos I found and a VS Code tip I wanted to share. I think it'll be helpful for you while working on a lot of duplicate content :blush: 

Go ahead and pass this to the next reviewer after fixing the typos!

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+Pass - Fail quality check
+=========================
+
+.. |QCP| replace:: :abbr:`QCP (Quality Control Point)`
+.. |QCPs| replace:: :abbr:`QCP (Quality Control Points)`
+
+In Odoo *Quality*, a *Pass - Fail* check is one of the quality check types that can be selected when
+creating a new quality check or quality control point (QCP). *Pass - Fail* checks consist of a text
+field that allows the creator to specify a certain criteria that a product must meet to pass the
+check.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:11]:
```suggestion
field that allows the creator to specify a certain criteria that a product must meet in order to
pass the check.
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+
+Create a Pass - Fail quality check
+==================================
+
+There are two distinct ways that *Pass - Fail* quality checks can be created. A single check can be
+manually created. Alternatively, a |QCP| can be configured that automatically creates checks at a
+predetermined interval.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:18]:
```suggestion
There are two distinct ways that *Pass - Fail* quality checks can be created.

A single check can be manually created, or a |QCP| can be configured that automatically creates
checks at a predetermined interval.
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+
+This documentation only details the configuration options that are unique to *Pass - Fail* quality
+checks and |QCPs|. For a full overview of all the configuration options available when creating a
+single check or a |QCP|, see the documentation on :ref:`quality checks
+<quality/quality_management/quality-checks>` and :ref:`quality control points
+<quality/quality_management/quality-control-points>`.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:24]:
```suggestion
This documentation only details the configuration options that are unique to *Pass - Fail* quality
checks and |QCPs|.

For a full overview of all the configuration options available when creating a single check or a
|QCP|, see the documentation on :ref:`quality checks <quality/quality_management/quality-checks>`
and :ref:`quality control points <quality/quality_management/quality-control-points>`.
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+
+To create a single *Pass - Fail* quality check, navigate to :menuselection:`Quality --> Quality
+Control --> Quality Checks`, and click :guilabel:`New`. Fill out the new quality check form as
+follows:
+
+- In the :guilabel:`Type` drop-down field, select the :guilabel:`Pass - Fail` quality check type.
+- In the :guilabel:`Team` drop-down field, select the quality team responsible for managing the
+  check.
+- In the :guilabel:`Instructions` text field of the :guilabel:`Notes` tab, enter instructions for
+  how to complete the quality check and the criteria that must be met for the check to pass.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:37]:
```suggestion
  how to complete the quality check, along with the criteria that must be met for the check to pass.
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+
+To create a |QCP| that generates *Pass - Fail* quality checks automatically, begin by navigating to
+:menuselection:`Quality --> Quality Control --> Control Points`, and click :guilabel:`New`. Fill out
+the new |QCP| form as follows:
+
+- In the :guilabel:`Type` drop-down field, select the :guilabel:`Pass - Fail` quality check type.
+- In the :guilabel:`Team` drop-down field, select the quality team responsible for managing the
+  checks created by the |QCP|.
+- In the :guilabel:`Instructions` text field, enter instructions for how to complete the quality
+  check and the criteria that must be met for the check to pass.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:54]:
```suggestion
  check, along with the criteria that must be met for the check to pass.
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+
+On an order
+-----------
+
+To process a *Pass - Fail* quality check on an order, select a manufacturing order or inventory
+order (receipt, delivery, return, etc.), for which a check is required. Manufacturing orders can be
+selected by navigating to :menuselection:`Manufacturing --> Operations --> Manufacturing Orders`,
+and clicking on an order. Inventory orders can be selected by navigating to
+:menuselection:`Inventory`, clicking the :guilabel:`# To Process` button on an operation card, and
+selecting an order.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:85]:
```suggestion
order (receipt, delivery, return, etc.), for which a check is required.

Manufacturing orders can be selected by navigating to :menuselection:`Manufacturing --> Operations
--> Manufacturing Orders`, and clicking on an order.

Inventory orders can be selected by navigating to :menuselection:`Inventory`, clicking the
:guilabel:`# To Process` button on an operation card, and selecting an order.
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst [hunk 11]
```diff
@@ -0,0 +1,132 @@
...
+
+.. image:: pass_fail_check/pass-fail-check-pop-up.png
+   :align: center
+   :alt: A Pass - Fail quality check pop-up window on a manufacturing or inventory order.
+
+If a quality alert must be created, click the :guilabel:`Quality Alert` button that appears at the
+top of the manufacturing or inventory order after the check fails. Clicking :guilabel:`Quality
+Alert` opens a quality alert form on a new page. For a complete guide on how to fill out quality
+alert forms, view the documentation on :ref:`quality alerts
+<quality/quality_management/quality-alerts>`.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:103]:
```suggestion
Alert` opens a quality alert form on a new page.

For a complete guide on how to fill out quality alert forms, view the documentation on :ref:`quality
alerts <quality/quality_management/quality-alerts>`.
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst [hunk 12]
```diff
@@ -0,0 +1,132 @@
...
+
+On a work order
+---------------
+
+When configuring a |QCP| that is triggered during manufacturing, a specific work order can also be
+specified in the :guilabel:`Work Order Operation` field on the |QCP| form. If a work order is
+specified, a *Pass - Fail* quality check is created for that specific work order, rather than the
+manufacturing order as a whole.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:111]:
```suggestion
manufacturing order, as a whole.
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst [hunk 13]
```diff
@@ -0,0 +1,132 @@
...
+
+*Pass - Fail* quality checks configured for work orders must be processed from the tablet view. To
+do so, begin by navigating to :menuselection:`Manufacturing --> Operations -->
+Manufacturing Orders`. Select a manufacturing order that includes a work order for which a quality
+check is required. Open the tablet view for that work order by selecting the :guilabel:`Work Orders`
+tab, and then clicking the :guilabel:`📱 (tablet)` button on the order's line.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:117]:
```suggestion
*Pass - Fail* quality checks configured for work orders **must** be processed from the tablet view.

To do so, begin by navigating to :menuselection:`Manufacturing --> Operations --> Manufacturing
Orders`. Then, select a manufacturing order that includes a work order for which a quality
check is required.

Open the tablet view for that work order by selecting the :guilabel:`Work Orders` tab, and clicking
the :guilabel:`📱 (tablet)` button on the order's line.
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst [hunk 14]
```diff
@@ -0,0 +1,132 @@
...
+
+With tablet view open, complete the steps listed on the left side of the screen until the *Pass -
+Fail* quality check step is reached. Upon reaching the check, follow the instructions that appear at
+the top of the screen. If the criteria for the check is met, click the :guilabel:`Pass` button at
+the top-right of the screen. If the criteria is not met, click the :guilabel:`Fail` button.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:122]:
```suggestion
With the tablet view open, complete the steps listed on the left side of the screen until the *Pass
- Fail* quality check step is reached. Upon reaching the check, follow the instructions that appear
at the top of the screen.

If the criteria for the check is met, click the :guilabel:`Pass` button at the top-right of the
screen. If the criteria is not met, click the :guilabel:`Fail` button.
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst [hunk 15]
```diff
@@ -0,0 +1,132 @@
...
+
+.. image:: pass_fail_check/work-order-pass-fail-check.png
+   :align: center
+   :alt: A Pass - Fail check for a manufacturing work order.
+
+If a quality alert must be created, do so by clicking the :guilabel:`☰ (menu)` button in the tablet
+view, and selecting :guilabel:`Quality Alert` from the :guilabel:`Menu` pop-up window. A
+:guilabel:`Quality Alerts` pop-up window appears, from which a quality alert can be created. For a
+complete guide on how to fill out quality alert forms, view the documentation on :ref:`quality
+alerts <quality/quality_management/quality-alerts>`.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:132]:
```suggestion
:guilabel:`Quality Alerts` pop-up window appears, from which a quality alert can be created.

For a complete guide on how to fill out quality alert forms, view the documentation on :ref:`quality
alerts <quality/quality_management/quality-alerts>`.
```

**Review by ksc-odoo — APPROVED** [review]
@hojo-odoo great work on this doc! once you give these suggested edits a look, and implement the necessary adjustments, I think this one will be ready for the next stage. nicely done 👍

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst [hunk 16]
```diff
@@ -0,0 +1,135 @@
...
+When configuring a |QCP| that is triggered during manufacturing, a specific work order can also be
+specified in the :guilabel:`Work Order Operation` field on the |QCP| form. If a work order is
+specified, a *Pass - Fail* quality check is created for that specific work order, rather than the
+manufacturing order as a whole.
+
+*Pass - Fail* quality checks configured for work orders must be processed from the tablet view. To
+do so, begin by navigating to :menuselection:`Manufacturing --> Operations -->
+Manufacturing Orders`. Select a manufacturing order that includes a work order for which a quality
+check is required. Open the tablet view for that work order by selecting the :guilabel:`Work Orders`
+tab, and then clicking the :guilabel:`📱 (tablet)` button on the order's line.
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:118]: Early line break
```suggestion
do so, begin by navigating to :menuselection:`Manufacturing --> Operations --> Manufacturing
Orders`. Select a manufacturing order that includes a work order for which a quality check is
required. Open the tablet view for that work order by selecting the :guilabel:`Work Orders` tab, and
then clicking the :guilabel:`📱 (tablet)` button on the order's line.
```

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @hojo-odoo! Nice work on this PR 🙂 I have only one small change after my technical review. I will push up this change after this review to move this PR forward!

Thank you!

**Comment by samueljlieber** [comment]
Implemented my change in https://github.com/odoo/documentation/commit/d2a4caa6c60ee9f1cc15a145b58482ba32c2b653. This PR is all set @StraubCreative 👍

**Comment by StraubCreative** [comment]
@robodoo r+

**Pull request closed without merging**